examples/TF_1_x/README.md

```diff
@@ -178,4 +178,6 @@ $ python3 examples/TF_1_x/object_detection_sample.py --input=examples/data/peopl
 
 For using camera as input use ```--input=0```. Here '0' refers to the camera present at /dev/video0. If the camera is connected to a different port, change it appropriately.
 
-**Note:** The results with input as an image or a directory of images, are written to output images. For video or camera input use the application display window for the results.
\ No newline at end of file
+**Note:** The results with input as an image or a directory of images, are written to output images. For video or camera input use the application display window for the results.
```

Comment:
We can remove these doc changes from this PR as this is a part of another open PR

Reply:
removed in this PR.